src/Directives/Php/PropertyDirective.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace T3Docs\GuidesPhpDomain\Directives\Php;
+
+use phpDocumentor\Guides\Nodes\CollectionNode;
+use phpDocumentor\Guides\Nodes\Node;
+use phpDocumentor\Guides\ReferenceResolvers\AnchorReducer;
+use phpDocumentor\Guides\RestructuredText\Directives\SubDirective;
+use phpDocumentor\Guides\RestructuredText\Parser\BlockContext;
+use phpDocumentor\Guides\RestructuredText\Parser\Directive;
+use phpDocumentor\Guides\RestructuredText\Parser\Productions\Rule;
+use phpDocumentor\Guides\RestructuredText\TextRoles\GenericLinkProvider;
+use Psr\Log\LoggerInterface;
+use T3Docs\GuidesPhpDomain\Nodes\MemberNameNode;
+use T3Docs\GuidesPhpDomain\Nodes\PhpConstNode;
+use T3Docs\GuidesPhpDomain\Nodes\PhpPropertyNode;
+use T3Docs\GuidesPhpDomain\PhpDomain\ModifierService;
+
+final class PropertyDirective extends SubDirective
+{
+    /**
+     * @var string[]
+     */
+    private array $allowedModifiers = ['public', 'protected', 'private', 'static', 'readonly'];
+
+    /** @var list<list<string>>  */
+    private array $illegalCombinations = [
+        ['private', 'protected'],
+        ['private', 'public'],
+        ['protected', 'public'],
+        ['protected', 'public'],
+        ['static', 'readonly'],
+    ];
+    public function __construct(
+        Rule $startingRule,
+        GenericLinkProvider $genericLinkProvider,
+        private readonly AnchorReducer $anchorReducer,
+        private readonly LoggerInterface $logger,
+        private readonly ModifierService $modifierService,
+    ) {
+        parent::__construct($startingRule);
+        $genericLinkProvider->addGenericLink($this->getName(), $this->getName());
+    }
+
+    public function getName(): string
+    {
+        return 'php:property';
+    }
+    /**
+     * @return string[]
+     */
+    public function getAliases(): array
+    {
+        return [
+            'php:attr',
```

Comment:
Should we add a log entry that "php:attr" is deprecated? AFAIR we did that also with "php:staticmethod"?

Reply:
I have no mechanism yet to deprecate aliases, though it would make absolutely sense to have that. I think there is something needed in the guides to make this

Reply:
Just a logger->notice or something?

Reply:
Ah okay, so it is not possible by now. Thought there is maybe some variable/method to get this information from.

Reply:
Found a way to do it, the property is called "name", maybe a little misfortune name but not sure what would be a better one.